unix/src/machine_stats/__init__.py

```diff
@@ -408,6 +466,14 @@ def main():
         nargs="*",
     )
 
+    measurement_args = parser.add_argument_group("Measurements arguments")
```

Comment:
To make it consistent with other arg groups: `positional arguments`, `optional arguments`
```suggestion
    measurement_args = parser.add_argument_group("measurements arguments")
```